test/SpokePoolClient.ValidateFill.ts

```diff
@@ -111,13 +110,53 @@ describe("SpokePoolClient: Fill Validation", function () {
     // "reasonable" block number based off the block time when looking at quote timestamps. We only need to do
     // this on the deposit chain because that chain's spoke pool client will have to fill in its realized lp fee %.
     await spokePool_1.setCurrentTime(await getLastBlockTime(spokePool_1.provider));
+
+    inputToken = erc20_1.address;
+    inputAmount = toBNWei(1);
+    outputToken = erc20_2.address;
+    outputAmount = inputAmount.sub(bnOne);
+  });
+
+  describe("v2", function () {
```

Comment:
OOC - is "v2" meant to be for backward compatibility?

Reply:
Yeah, but it has been removed now after we dropped v2 functions entirely from the relayer.